save_path_rel: "dataset/analysis/"

action_time_horizon: 6.0

goal_range_min_x: -5.0
goal_range_max_x: 5.0

goal_range_min_y: -2.5
goal_range_max_y: 2.0

goal_range_min_z: 0.5
goal_range_max_z: 1.0

## Reward:
reward_terminal_goal: 50
reward_terminal_collision: -50
reward_terminal_roll: -50
reward_terminal_max_step: -50

reward_step_target2goal: 10
reward_step_target2goal_threshold: 2
reward_step_target2goal_scale_beta: 1

reward_step_mode0: -0.9
reward_step_mode1: -1.8
reward_step_mode2: -2.7

reward_step_mpc_exit: -20

reward_step_time_horizon_min: -20
reward_step_time_horizon_max: 10

alpha_step_goal_base: 1
alpha_step_goal_ee: 1
alpha_step_target_pos: 1
alpha_step_target_ori: 1
alpha_step_mode: 1
alpha_step_mpc_exit: 1